### PR TITLE
feat(blueprint): Update Blueprint styles

### PR DESCRIPTION
**Pull request by diegographext**
This PR encompasses several changes in the style for different tasks.

**JIRAS** 
[GRAPH-6950](https://graphext.atlassian.net/browse/GRAPH-6950)
[GRAPH-6887](https://graphext.atlassian.net/browse/GRAPH-6887)
[GRAPH-7108](https://graphext.atlassian.net/browse/GRAPH-7108)
[GRAPH-6807](https://graphext.atlassian.net/browse/GRAPH-6807)

## Changes made:

- Adapted EditableText
- Add new **Custom Class** for EditableText with less border-radius
- Add EditableText examples (like "disabled" or "gp-rounded-small")
- Centered cells and truncated icon for DATA
- Created **Custom Class** for truncated cells (to show the icon only on hover)
- Adapted truncated popover style (font and white-space)
- Adapted table headers styles
- Adapted table selection styles
- Removed strange artefacts when selected on table headers
- New Design System colors in HSL with tint and shade functions.
- Button icons color (was 50% intensity and now is 100%)

## How it looks:

**Adapted EditableText**
Captura de pantalla 2022-03-28 a las 11 30 26

**Custom Class for EditableText with less border-radius**
Captura de pantalla 2022-03-28 a las 11 29 47

**EditableText examples**

https://user-images.githubusercontent.com/97446955/160380714-ecc6e211-9491-4540-a992-109f588ce268.mov

**Centered cells and truncated icon for DATA**
Captura de pantalla 2022-03-28 a las 11 33 10

**Custom Class for truncated cells (to show the icon only on hover)**

https://user-images.githubusercontent.com/97446955/160380289-8fc6ea25-72f5-4a94-9e4d-b6e75e8ca54c.mov

**Adapted truncated popover style (font and white-space)**
Captura de pantalla 2022-03-28 a las 11 35 42

**Adapted table headers and selection styles**
Before:

https://user-images.githubusercontent.com/97446955/160380989-2ddc0977-5458-48fb-ae58-1aff79c5c5ac.mov

After:

https://user-images.githubusercontent.com/97446955/160381070-471825b5-59bf-431c-9434-2cff402bb3cf.mov

### CODE REVIEW

File: packages/core/src/common/_customs.scss [hunk 1]
```diff
@@ -94,3 +94,58 @@ $pt-dark-divider-black: $white3;
 
 // Custom shadow for buttons, selects, tag-inputs, tooltips & datetime
 $custom-shadow: 0 0 12px rgba($black, 0.25);
+
+// Custom shadow for popover (New Design System)
+$pt-dark-border-shadow-opacity: 0.1;
+
+// ------------------------- New Design System Colors -------------------------------
```
Comment (diegographext) [inline packages/core/src/common/_customs.scss:101]: Defined 2 functions to get the new design systems colors (with tint and shade)

File: packages/core/src/components/button/_overrides.scss [hunk 2]
```diff
@@ -42,9 +42,27 @@
...
 .#{$ns}-button-group .#{$ns}-popover-wrapper:not(:last-child) .#{$ns}-button {
-  border-right: 1px solid #2A2E38; // --Dark3
+  border-right: 1px solid $g-dark-base3;
 }
 
 .#{$ns}-dark .#{$ns}-button-group .#{$ns}-button:not([class*="bp3-intent-"]) {
   box-shadow: none;
 }
+
+// Fix - Button icons without intent not 100% white
```
Comment (crispamares) [inline packages/core/src/components/button/_overrides.scss:52]: nice comment ^_^

File: packages/core/src/components/editable-text/_overrides.scss [hunk 3]
```diff
@@ -1,25 +1,43 @@
...
+    @include position-editable-text(-8px, -12px); // Should be the same as margin
+  }
+
+  // CUSTOM CLASS Small rounded for table description editable text
+  &.#{$ns}-gp-rounded-small {
+    margin: 8px 8px; // Forcing smaller margin
+
+    &::before {
+      border-radius: 8px;
+      @include position-editable-text(-8px, -8px); // Should be the saame as margin
```
Comment (crispamares) [inline packages/core/src/components/editable-text/_overrides.scss]:
```suggestion
      @include position-editable-text(-8px, -8px); // Should be the same as margin
```

**Review by crispamares — APPROVED** [review]
Superb comments !  With css it is easy to lost the purpose of the code. Love it.

Also... which is the goal of changing the Examples ? In my mind, the storybook was meant to this.

We used to avoid changes in tsx code in order to avoid conflicts when merging from upstream. I see the changes are not very big, and they are not changing the examples too often... but if you expect that we will introduce more and more examples we can think a solution like "custom" and "overwrite" to add "graphext's examples"

**Pull request merged**